algorithms/hhl/hhl/hhl.ipynb

```diff
@@ -1,982 +1,1292 @@
 {
```

Comment:
<p>This sentence is written twice:</p><p>"Note that the HHL algorithm returns a statevector result that includes an arbitrary global phase. The global phase can arise due to the transpilation process or from the intrinsic properties of the quantum functions themselves".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Duplicate removed.</p>